socorro/cron/jobs/upload_crash_report_json_schema.py

```diff
@@ -44,14 +44,34 @@ class UploadCrashReportJSONSchemaCronApp(BaseCronApp):
         default='crash_report.json',
         doc="Name of the file/key we're going to upload to"
     )
+    required_config.add_option(
+        'telemetry_bucket_name',
+        default='',
+        reference_value_from='resource.boto',
+        doc='if set, overrides resource_class bucket name'
+    )
+
+    def get_bucket_name(self):
```

Comment:
Should this be a `@property` instead of a method?

Reply:
I find properties convenient especially where `.set` needs to do interesting things or `.get` should get cached. Otherwise, I almost never use properties because they're a pain in the ass to subclass and override.